### PR TITLE
Drop custom appstream-compose command

### PR DESCRIPTION
**Pull request by grulja**

### CODE REVIEW

**Comment by grulja** [comment]
Hi @bbhtt, I see you have been helping with related failures. This fails with:
```
Errors were raised during this compose run:
general
  E: filters-but-no-output
```
Any idea what is wrong or missing in this case?

**Comment by bbhtt** [comment]
You can drop the compose step and `appstream-compose: false` in manifest toplevel

It works ootb now, there is no need for a custom compose command

**Comment by grulja** [comment]
> You can drop the compose step and `appstream-compose: false` in manifest toplevel
> 
> It works ootb now, there is no need for a custom compose command

Thank you. Trying it now.

File: org.kde.WaylandDecoration.QAdwaitaDecorations.json [hunk 1]
```diff
@@ -27,7 +26,6 @@
             "build-commands": [
                 "mkdir -p ${FLATPAK_DEST}/share/appdata",
                 "cp org.kde.WaylandDecoration.QAdwaitaDecorations.appdata.xml ${FLATPAK_DEST}/share/appdata",
```
Comment (bbhtt) [inline org.kde.WaylandDecoration.QAdwaitaDecorations.json]:
```suggestion
                "cp org.kde.WaylandDecoration.QAdwaitaDecorations.appdata.xml ${FLATPAK_DEST}/share/appdata"
```

Comment (grulja) [reply org.kde.WaylandDecoration.QAdwaitaDecorations.json]: Yeah, I realized I probably left a comma there after I saw the json is not valid, but thanks for also suggesting the fix.

**Pull request merged**